product_family/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# (c) 2017 Consultoría Informática Studio73 SL (contacto@studio73.es)
+#          Pablo Fuentes <pablo@studio73.es>
+#
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Product Family',
+    'summary': 'Product Family',
+    'author': 'Consultoría Informática Studio 73 S.L.',
```

Comment:
Modules shared to OCA should have Odoo community Association as co author. (see other modules in the same repository for exemple). this is usefull to see easily in the appstore, which module is under OCA umbrella.